tests/commands/hook_impl_test.py

```diff
@@ -97,6 +97,7 @@ def call(*_, **__):
         ('pre-push', ['branch_name', 'remote_name']),
         ('commit-msg', ['.git/COMMIT_EDITMSG']),
         ('post-commit', []),
+        ('post-merge', [True]),
```

Comment:
this doesn't match the arguments that git will send (just to make this realistic, these should all be strings)